edi_purchase_config/views/res_partner_view.xml

```diff
@@ -24,9 +24,9 @@
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="base.view_partner_form" />
         <field name="arch" type="xml">
-            <field name="category_id" position="after">
-                <field name="is_edi" />
-            </field>
+            <xpath expr="//form/sheet/notebook/page[@name='sales_purchases']/group[@name='container_row_2']/group[@name='purchase']/field[@name='supplier']" position="after">
+                <field name="is_edi"/>
+            </xpath>
```

Comment:
Please don't use a long xpath expr.
They will just be a pain in the ass with compatibility with other modules. Sometimes other modules would change an element or move it around and it will completely mess with your xpath expr.

 In fact try to avoid xpath in favour of good old `<field name="">` tags whenever possible.

What's wrong with

```xml
<field name="supplier" position="after">
    <field name="is_edi"/>
</field> 
```

?
 is it not working?
if not, I'm sure you can probably find another way either with less explicity xpath or finding another field to reference

ie:

```xml
<group name="purchase" position="inside">
. . .
</group>
```

that would also work, probably..